mountpoint-s3/src/cli.rs

```diff
@@ -615,10 +636,19 @@ pub fn create_s3_client(args: &CliArgs) -> anyhow::Result<(S3CrtClient, EventLoo
         user_agent.key_value("mp-cache-ttl", &ttl.to_string());
     }
 
+    let (read_part_size, write_part_size) = match (args.read_part_size, args.write_part_size) {
+        (None, None) => (args.part_size, args.part_size),
+        _ => (
+            args.read_part_size.unwrap_or(args.part_size),
+            args.write_part_size.unwrap_or(args.part_size),
+        ),
+    };
```

Comment:
I think we should put it back how you had it. We didn't gain anything by moving it out. (If we had changed `CliArgs::part_size` to `Option<u64>`, it would be different, but I don't think we should do that.

```rust
let mut client_config = S3ClientConfig::new()
        .auth_config(auth_config)
        .throughput_target_gbps(throughput_target_gbps)
        .read_part_size(args.read_part_size.unwrap_or(args.part_size) as usize)
        .write_part_size(args.write_part_size.unwrap_or(args.part_size) as usize)
        .user_agent(user_agent);
```

Reply:
Yeah, true, i forgot to changed it back when I drop option<u64>.